src/develop/tutorial/smart-digital-assets/issue-ft.md

```diff
@@ -1,15 +1,508 @@
+deployWalletValue: locklift.utils.toNano(1),
+mintDisabled: disableMint,
+burnByRootDisabled: disableBurnByRoot,
+burnPaused: pauseBurn,
+remainingGasTo: new Address(myAccountAddress),
+},
+value: locklift.utils.toNano(5),
+});
+console.log(`${name}: ${tokenRoot.address}`);
+}
+main()
+.then(() => process.exit(0))
+.catch(e => {
+console.log(e);
+process.exit(1);
+});
+```
+
+You can then run the script using the following Locklift command:
+
+```shell
+npx locklift run -s ./scripts/01-deploy-token.ts -n local
+```
+
+After running the script, your TIP-3 Token Root will be deployed, and in the terminal you will see token data:
+
+```shell
+...
+Root owner ...
+Name ...
+Symbol ...
+Decimals ...
+...
+```
+
+## Deploy Token Wallet
+
+Let's write the script `/scripts/02-deploy-wallet.js` for Token Wallet deployment.
+
+```ts
+import { Address, Contract } from "locklift/.";
+import { AccountAbi } from "../build/factorySource";
+async function main() {
+  const tokenRootAddress = "";
+  const myAccountAddress = "";
+
+  const signer = (await locklift.keystore.getSigner("0"))!;
+  const accountFactory = locklift.factory.getAccountsFactory("Account");
+
+  /*
+    Get Account contract instance: Account<AccountAbi>
+  */
+  const account = accountFactory.getAccount(myAccountAddress, signer.publicKey);
+
+  /*
+    Get instance of already deployed contract
+  */
+  const tokenRoot = locklift.factory.getDeployedContract(
+    "TokenRoot",
+    new Address(tokenRootAddress),
+  );
+
+  /*
+    Under the hood of runTarget, the Account sendTransaction method is called
+  */
+  await account.runTarget(
+    {
+      contract: tokenRoot,
+      value: locklift.utils.toNano(1),
+    },
+    tr =>
+      tr.methods.deployWallet({
+        answerId: 0,
+        walletOwner: account.address,
+        deployWalletValue: 0,
+      }),
+  );
+
+  /*
+    We call the get method on the contract.
+    @params answerId: If the function marked as responsible then
+      field answerId appears in the list of input parameters
+        of the function in *abi.json file. answerId is function id
+        that will be called.
+      In external calls, we did not notice that this affected anything.
+        But answerid is critical when you call these functions on-chain
+  */
+  const wallet = await tokenRoot.methods
+    .walletOf({
+      answerId: 0,
+      walletOwner: account.address,
+    })
+    .call({ responsible: true });
+  console.log(`Account deployed at: ${Account.address}`);
+  console.log(`TIP3 Wallet deployed at: ${wallet.value0.toString()}`);
+}
+main()
+  .then(() => process.exit(0))
+  .catch(e => {
+    console.log(e);
+    process.exit(1);
+  });
+```
+
+[The TON Soidity Compiler](https://github.com/tonlabs/TON-Solidity-Compiler/blob/master/API.md#addresstransfer) allows you to specify different parameters (value,currencies, bounce or flag) of an outgoing internal message sent through an external function call.
```

Comment:
This text is taken out of context and confuses the reader more than it explains anything. If we want to talk about the `responsible` function, then we need to make a link to the documentation for this modifier.

Reply:
https://github.com/everscale-org/docs/blob/9d8035652d0f049c8184d4effc8f588e79d7123e/src/develop/tutorial/smart-digital-assets/issue-ft.md?plain=1#L355